cpp/sopt/l1_proximal.h

```diff
@@ -63,6 +67,8 @@ class L1TightFrame {
   SOPT_MACRO(Psi, LinearTransform<Vector<Scalar>>);
   //! Bound on the squared norm of the operator Ψ
   SOPT_MACRO(nu, Real);
+  SOPT_MACRO(Phi, LinearTransform<Vector<Scalar>>);
+  SOPT_MACRO(beta, Real);
```

Comment:
The `SOPT_MACRO`s above have the `//!` comments explaining them, should we add comments on these two new ones as well?

Reply:
Yes, good point. I can't find any description of `beta` other than "Beta parameter" so I'm just copying it here, with a note of why it's being added.